src/_assets/css/_dash.scss

```diff
@@ -179,11 +198,25 @@ $dash-dartpad-border: #293542;
 
   .bullet-text{
     margin-left: 20px;
+
+    // Disable text selection to avoid blue focus outline on macOS
```

Comment:
Does this reduce accessibility? (Also, selfishly, I often copy text, so anything that makes that impossible worries me.)